test/e2e/service.go

```diff
@@ -1151,6 +1171,31 @@ func testReachable(ip string, port int) bool {
 	return testReachableInTime(ip, port, podStartTimeout)
 }
 
+func testNetcatReachable(ip string, port int) {
+	con, err := net.Dial("udp", ip+":"+string(port))
+	if err != nil {
+		Failf("Failed to connect to: %s:%d (%s)", ip, port, err.Error())
+	}
+
+	_, err = con.Write([]byte("\n"))
+	if err != nil {
+		Failf("Failed to send newline: %s", err.Error())
+	}
+
+	var buf []byte = make([]byte, len("SUCCESS")+1)
```

Comment:
Yes, but at the end we'll get user reports that say: hey I tried lb with 1mb data grams and everything came to a grinding halt, maybe it's kube-proxy? and we'd say, aha- it's a gce limitation, and waste no more time ;)